scripts/github-export.py

```diff
@@ -0,0 +1,132 @@
+"""
+poetry init -n
+poetry add requests
```

Comment:
This should be moved to `pyproject.tom` as dev dependency. And whole this comment should be removed.

Reply:
Comments removed.